assets/css/common/post-single.css

```diff
@@ -267,6 +267,7 @@
     background: var(--code-bg);
     border-radius: var(--radius);
     padding: .4em
```

Comment:
Missing semicolon. This messes up the padding of TOC elements.

Reply:
I thought as much. This was what came to mind
Thanks.